DOCS/man/osc.rst

```diff
@@ -315,6 +315,11 @@ Configurable Options
 
     Display timecodes with milliseconds
 
+``tcwidth``
+    Logical width of timecode elements, to compensate for wider or narrower
+    fonts. 100 represents the default width, and values smaller than 50 or
+    larger than 200 are ignored.
```

Comment:
"Default: 100" should come first - like elsewhere.

The bigger issue though is that it doesn't explain what it's good for. The only reason this option exists is to adjust the spacing around the two timecode values due to wider-than-expected font, but I don't think a reasonable user could guess that from this documentation.

I think something along these lines could be better (rephrase/adjust as you see fit):

Adjust the spacing around the timecode values. The timecode width depends on the font, and with some fonts the spacing near the timecodes becomes too small. Use values above 100 to increase that spacing, or below 100 to decrease it (allowed: 50-200).

(I wrote this before you edited this documentation, but my comment still largely apply)

Reply:
Also, no need for the co-authoring thing. Keep it as your commit (if you agree with it. if you don't then remove it from the PR and I'll push it myself as my own).